web/app/view/MapMarkerController.js

```diff
@@ -488,5 +494,56 @@ Ext.define('Traccar.view.MapMarkerController', {
         } else if (point) {
             this.getView().getMapView().fit(new ol.geom.Point(point));
         }
+    },
+
+    updateVisibleDevice: function (store, data) {
```

Comment:
Does this method ever gets array? I guess then it would make sense to rename it to plural "devices".